hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -509,20 +507,20 @@ def accept(f):
         elif argspec.kwonlyargs:
             parts.append("*")
         for k in must_pass_as_kwargs:
-            invocation_parts.append("%(k)s=%(k)s" % {"k": k})
+            invocation_parts.append("{k}={k}".format(k=k))
```

Comment:
Hm, it seems like an f-string should work here (and on the line 513)? `f"{k}={k}"`